App.tsx

```diff
@@ -46,7 +46,7 @@ const App = () => {
         <PersistGate loading={null} persistor={persistor}>
           <NavigationContainer>
             <Stack.Navigator
-              initialRouteName="Landing"
+              initialRouteName="Staking"
```

Comment:
I think this was for testing?

Reply:
yes, with the commit I've just pushed it's "Landing" again